src/commonMain/kotlin/org/hisp/dhis/rules/models/RuleLocalDate.kt

```diff
@@ -0,0 +1,31 @@
+package org.hisp.dhis.rules.models
+
+import kotlinx.datetime.LocalDate
+import kotlin.js.JsExport
+import kotlin.jvm.JvmStatic
+
+@JsExport
+data class RuleLocalDate(val year: Int, val month: Int, val day: Int) {
+    private val localDate: LocalDate = LocalDate.parse("$year-${month.toString().padStart(2, '0')}-${day.toString().padStart(2, '0')}");
```

Comment:
The class `LocalDate` can be instantiated as `LocalDate(year, month, day)`, wouldn't it be better to use this constructor?